src/OrchardCore/OrchardCore.Abstractions/Localization/ILocalizationDataProvider.cs

```diff
@@ -0,0 +1,9 @@
+using System.Collections.Generic;
+
+namespace OrchardCore.Localization
+{
+    public interface ILocalizationDataProvider
+    {
+        IEnumerable<DataLocalizedString> GetAllStrings();
```

Comment:
This is a bad name. More like GetDescriptors. It's confusing with the GetAllStrings of the data localizer